src/rules/noMagicNumbersRule.ts

```diff
@@ -70,6 +71,10 @@ export class Rule extends Lint.Rules.AbstractRule {
 class NoMagicNumbersWalker extends Lint.AbstractWalker<Set<string>> {
     public walk(sourceFile: ts.SourceFile) {
         const cb = (node: ts.Node): void => {
+            if (isCallExpression(node) && isIdentifier(node.expression) && node.expression.text === "parseInt") {
+                return;
```

Comment:
you probably want to `return node.arguments.length === 0 ? undefined : cb(node.arguments[0]);`
Otherwise magic numbers in the first argument are not checked.

You can also add a test to ensure it works as expected:
```
parseInt(foo === 4711 ? bar : baz, 10);
                 ~~~~ [error message]
```

And another test to ensure it doesn't crash if there are no arguments at all:
```ts
parseInt();
```